app/helpers/interruption_screen_helper.rb

```diff
@@ -7,6 +7,8 @@ def display_informative_text(informative_text, lettings_log)
       value = if argument["label"]
                 pre_casing_value = lettings_log.form.get_question(argument["key"], lettings_log).answer_label(lettings_log)
                 pre_casing_value.downcase
+              elsif argument["currency"]
+                ["£", number_to_currency(lettings_log.public_send(argument["key"]), delimiter: ",", format: "%n")].join("")
```

Comment:
Could it not be a label true and a currency? Or a label false and currency?

Reply:
I don't necessarily see the need for the currency keyword.
If it's a currency and we're getting it from the label, that should set the "£" for us and format it in the way that the answer would be displayed in the answer options. If it's not a label, that means it's most likely a method, so you could define the formatting in that method that's specific to your error

Reply:
I guess it felt like this should already be happening somewhere and we wouldn't have to explicitly define it, but happy for this to go in if it's more readable and makes more sense!